src/main/java/frc/robot/RobotContainer.java

```diff
@@ -146,14 +141,14 @@ private void configureButtonBindings() {
     m_driverController.rightStick().whileTrue(new setXCommand());
     m_driverController.leftTrigger().whileTrue(new IntakeNote());
     m_driverController.rightTrigger().whileTrue(new EmptyStomach()).onFalse(new ResetWheels());
-    m_driverController.y().onTrue(new PivotPID(SmartDashboard.getNumber("Test Pivot", ManipulatorConstants.PIVOT_MAX)));
+    m_driverController.y().onTrue(new PivotPID());
     m_driverController.a().whileTrue(new PivotDown(0.5, -0.1));
     m_driverController.start().whileTrue(new ResetRobotHeading());
     m_driverController.x().whileTrue(new TurnToAngle(90));
     m_driverController.b().whileTrue(new TurnToAngle(-90));
 
-    m_coDriverController.rightStick().whileTrue(new setXCommand());
-    m_coDriverController.rightTrigger().onTrue(new SimpleScoreNote(ManipulatorConstants.PIVOT_AMP_GOAL, 2000));
+    m_coDriverController.rightStick().whileTrue(new SimpleScoreTest());
```

Comment:
so this is a `whileTrue` instead of an `onTrue` which may cause some problems, don't use the sticks as a button, use I think the codriver `a` and `x` buttons are free so use one of those.